tasks/us/census/acs.py

```diff
@@ -3250,12 +3155,124 @@ def columns(self):
+            ('male_45_64_grade_9_12', male_45_64_grade_9_12),
+            ('male_45_64_high_school', male_45_64_high_school),
+            ('male_45_64_some_college', male_45_64_some_college),
+            ('male_45_to_64', male_45_to_64),
+            ('male_male_households', male_male_households),
+            ("management_business_sci_arts_employed", management_business_sci_arts_employed),
             ('no_car', no_car),
+            ('no_cars', no_cars),
+            ("not_us_citizen_pop", not_us_citizen_pop),
+            ('occupation_management_arts', occupation_management_arts),
+            ('occupation_natural_resources_construction_maintenance',
+             occupation_natural_resources_construction_maintenance),
+            ('occupation_production_transportation_material',
+             occupation_production_transportation_material),
+            ('occupation_sales_office', occupation_sales_office),
+            ('occupation_services', occupation_services),
+            ('one_car', one_car),
+            ('two_cars', two_cars),
+            ('three_cars', three_cars),
+            ("pop_25_64", pop_25_64),
+            ("pop_determined_poverty_status", pop_determined_poverty_status),
+            ('population_1_year_and_over', population_1_year_and_over),
+            ("population_3_years_over", population_3_years_over),
+            ("poverty", poverty),
+            ("sales_office_employed", sales_office_employed),
+            ('some_college_and_associates_degree', some_college_and_associates_degree),
+            ("walked_to_work", walked_to_work),
+            ("worked_at_home", worked_at_home),
+            ("workers_16_and_over", workers_16_and_over),
+            ("associates_degree", associates_degree),
+            ("bachelors_degree", bachelors_degree),
+            ("high_school_diploma", high_school_diploma),
+            ("less_one_year_college", less_one_year_college),
+            ("masters_degree", masters_degree),
+            ("one_year_more_college", one_year_more_college),
+            ("pop_25_years_over", pop_25_years_over),
+            ("commute_35_44_mins", commute_35_44_mins),
+            ("commute_60_more_mins", commute_60_more_mins),
+            ("commute_less_10_mins", commute_less_10_mins),
+            ("commuters_16_over", commuters_16_over),
+            ('hispanic_any_race', hispanic_any_race),
+            ("pop_5_years_over", pop_5_years_over),
+            ("speak_only_english_at_home", speak_only_english_at_home),
+            ("speak_spanish_at_home", speak_spanish_at_home),
+            ('speak_spanish_at_home_low_english', speak_spanish_at_home_low_english),
+            ("pop_15_and_over", pop_15_and_over),
+            ("pop_never_married", pop_never_married),
+            ("pop_now_married", pop_now_married),
+            ("pop_separated", pop_separated),
+            ("pop_widowed", pop_widowed),
+            ("pop_divorced", pop_divorced),
```

Comment:
This is ok for now, but in the future we can probably migrate all of this to the `columns.json` file. It is quite easy to autogenerate (basically, iterate over this object and generate JSON about it). The only trick is to handle the references between columns (but we can copy the approach that I use in the UK thingy).

Anyway, we can leave it as it is for now, as this PR is getting big, and revisit in the future.

Reply:
That is the idea, we should merge the `columns.py` from this PR with [your column generation through definition](https://github.com/CartoDB/bigmetadata/blob/ab24f3b52b67fc601991d5a7d3f980dc15fa368c/tasks/uk/census/metadata.py#L68-L89). This would be the first step towards declarative definition of columns.

Reply:
My idea with UK was to serve as a prototype for now. It has some ugly things that are not ideal for reuse (for example, the way it handles summing of fields, which is a feature which probably is not useful for other countries).

I'm starting a document to talk about this, and see what parts we can do with a generic approach (probably, metadata generation) and which need to be done with per-country extensions.